packages/core/saved-objects/core-saved-objects-base-server-internal/src/validation/validator.ts

```diff
@@ -44,7 +44,7 @@ export class SavedObjectsTypeValidator {
     this.validationMap = typeof validationMap === 'function' ? validationMap() : validationMap;
     this.orderedVersions = Object.keys(this.validationMap).sort(Semver.compare);
   }
-
+  // uses index meta's mappingVersions & docVersions -> virtualVersion to allow keeping track of mixed stack and model versions per type.
```

Comment:
🧡 

nit: should it be `/**` comment so it builds JSDocs?

Reply:
I got confused with the `public validate`. But I understand that it's not exposed publicly outside the package. While JSDocs might help because the IDE would show the explanation of the function where used, it's not a big deal to keep it as-is :)

Reply:
@afharo I uncovered extra issues with this work and it's not as "simple" as originally thought and I'll handle the nits while doing that.

Reply:
I removed the comment.